Umbraco-Heartcore/Versions-and-updates/index.md

```diff
@@ -0,0 +1,17 @@
+---
+versionFrom: 8.0.0
+---
+
+# Versions and updates
+
+Umbraco Heartcore is a headless SaaS (software as a service) product, which means that one of the benefits is that you do not have to worry about upgrades. These will all be handled by Umbraco HQ. 
+
+Umbraco HQ will regularly be adding new features and endpoints to the REST APIs, as well as maintaining the existing structure and endpoints. Each time this is done, a new version of the API will be released. API releases and versioning is handled following the [ASP.NET API versioning guidelines](https://github.com/microsoft/aspnet-api-versioning).
```

Comment:
Umbraco HQ will regularly be adding new features to the cms and new capabilities to the REST APIs. 
Each time new endpoints are added to the REST API the `api-version` will be updated to reflect the changes. API releases and versioning is handled following the [ASP.NET API versioning guidelines](https://github.com/microsoft/aspnet-api-versioning). We follow semver in terms of versioning, so you can be sure that existing APIs will not break and continue to work as expected. If a breaking change is required then the major version will be incremented, but the previous major version will continue to work.